library/src/com/astuetz/PagerSlidingTabStrip.java

```diff
@@ -194,8 +189,11 @@ public PagerSlidingTabStrip(Context context, AttributeSet attrs, int defStyle) {
         isPaddingMiddle = a.getBoolean(R.styleable.PagerSlidingTabStrip_pstsPaddingMiddle, isPaddingMiddle);
         tabTypefaceStyle = a.getInt(R.styleable.PagerSlidingTabStrip_pstsTextStyle, Typeface.BOLD);
         tabTypefaceSelectedStyle = a.getInt(R.styleable.PagerSlidingTabStrip_pstsTextSelectedStyle, Typeface.BOLD);
-        tabTextAlpha = a.getFloat(R.styleable.PagerSlidingTabStrip_pstsTextAlpha, HALF_TRANSP);
-        tabTextSelectedAlpha = a.getFloat(R.styleable.PagerSlidingTabStrip_pstsTextSelectedAlpha, OPAQUE);
+        tabTextColorSelected = a.getColorStateList(R.styleable.PagerSlidingTabStrip_pstsTextColorSelected);
+        if (tabTextColorSelected == null) {
+            tabTextColorSelected = tabTextColor;
+        }
+
```

Comment:
Of course! Sorry, I started to through ideas into the code and I end up with this. I didn´t even check the Color API. After refactor it would be something like that. Everyone ok with that?

```xml
   <attr name="pstsTextAlpha" format="integer" />

   <com.astuetz.PagerSlidingTabStrip
        android:id="@+id/tabs"
        android:layout_width="match_parent"
        android:layout_height="?attr/actionBarSize"
        android:background="?attr/colorPrimary"
        app:pstsTextAlpha="150"/> <!--Range 0..255->
```
```java
        ColorStateList colorStateList = a.getColorStateList(TEXT_COLOR_INDEX);
        tabTextColorSelected = a.getColorStateList(R.styleable.PagerSlidingTabStrip_pstsTextColorSelected);
        textAlpha = a.getInt(R.styleable.PagerSlidingTabStrip_pstsTextAlpha, alpha);

        if (colorStateList != null) {
            tabTextColor = colorStateList;
        } else {
            int textPrimaryColorTransp = Color.argb(textAlpha,
                    Color.red(textPrimaryColor),
                    Color.green(textPrimaryColor),
                    Color.blue(textPrimaryColor));
            tabTextColor = getColorStateList(textPrimaryColorTransp);
        }

        if (tabTextColorSelected == null) {
            tabTextColorSelected = getColorStateList(textPrimaryColor);
        }
```